monai/bundle/scripts.py

```diff
@@ -621,3 +624,80 @@ def ckpt_export(
         more_extra_files=extra_files,
     )
     logger.info(f"exported to TorchScript file: {filepath_}.")
+
+
+def init_bundle(
+    bundle_dir: PathLike,
+    ckpt_file: Optional[PathLike] = None,
+    network: Optional[torch.nn.Module] = None,
+    metadata_str=DEFAULT_METADATA,
+    inference_str=DEFAULT_INFERENCE,
+):
+    """
+    Initialise a new bundle directory with some default configuration files and optionally network weights.
+
+    Typical usage example:
+
+    .. code-block:: bash
+
+        python -m monai.bundle init_bundle /path/to/bundle_dir network_ckpt.pt
+
+    Args:
+        bundle_dir: directory name to create, must not exist but parent direct must exist
+        ckpt_file: optional checkpoint file to copy into bundle
+        network: if given instead of ckpt_file this network's weights will be stored in bundle
+    """
+
+    bundle_dir = Path(bundle_dir).absolute()
+
+    if bundle_dir.exists():
+        raise ValueError(f"Specified bundle directory '{str(bundle_dir)}' already exists")
+
+    if not bundle_dir.parent.is_dir():
+        raise ValueError(f"Parent directory of specified bundle directory '{str(bundle_dir)}' does not exist")
+
+    configs_dir = bundle_dir / "configs"
+    models_dir = bundle_dir / "models"
+    docs_dir = bundle_dir / "docs"
+
+    bundle_dir.mkdir()
+    configs_dir.mkdir()
+    models_dir.mkdir()
+    docs_dir.mkdir()
+
+    if isinstance(metadata_str, dict):
+        metadata_str = json.dumps(metadata_str, indent=4)
+
+    if isinstance(inference_str, dict):
+        inference_str = json.dumps(inference_str, indent=4)
+
+    with open(str(configs_dir / "metadata.json"), "w") as o:
+        o.write(metadata_str)
+
+    with open(str(configs_dir / "inference.json"), "w") as o:
+        o.write(inference_str)
+
+    with open(str(docs_dir / "README.md"), "w") as o:
+        readme = """
+        # Your Model Name
+
+        Describe your model here and how to run it, for example using `inference.json`:
+
+        ```
+        python -m monai.bundle run evaluating \
+            --meta_file /path/to/bundle/configs/metadata.json \
+            --config_file /path/to/bundle/configs/inference.json \
+            --dataset_dir ./input \
+            --bundle_root /path/to/bundle
+        ```
+        """
+
+        o.write(dedent(readme))
+
+    with open(str(docs_dir / "license.txt"), "w") as o:
+        o.write("Select a license and place its terms here\n")
+
+    if ckpt_file is not None:
+        copyfile(str(ckpt_file), str(models_dir / "model.pt"))
+    elif network is not None:
+        torch.save(get_state_dict(network), str(models_dir / "model.pt"))
```

Comment:
I would suggest to use `save_state()` API instead:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/networks/utils.py#L471
It has more flexible support.

Thanks.